### ISSUE TITLE
tiny: visitor profile: linebreak between value and unit

### ISSUE THREAD

**Issue by hpvd** [piwik/piwik#7608]
tiny: visitor profile: linebreak between value and unit
![2015-04-01_10h53_41](https://cloud.githubusercontent.com/assets/5681880/6937804/ae8bb0f2-d85e-11e4-9989-03e633f78cb3.png)

please see attachment

**Comment by saleemkce** [comment]
It has nothing to do with line break because it always depends on screen resolution/size. Even if you use "<br/>UNIT VALUE" pattern, for smaller or wider resolutions, it will put this pattern in new line and would seem like a TEXT break. I have faced this issue too many times in the past. One possible solution would be like using a UI label like this 
https://github.com/piwik/piwik/labels/c:%20Design%20/%20UI
 which will ensure nice UI and automatically prevents line breaks for any "UNIT VALUE" patterns.

**Comment by hpvd** [comment]
sounds good.
Maybe one should collect all cases/places where this could happen within this ticket?

From image above one could see that these are not only simple ones "UNIT VALUE" but also "VALUE1 UNIT1 VALUE2 UNIT2" e.g. for durations like "3 hours 29 min"

**Comment by saleemkce** [comment]
Yes, that's exactly what I too thought. We need to fix this in all the occurrences possible so that uniformity is maintained throughout the site and such key metrics is highlighted by default.

**Comment by mnapoli** [comment]
It could also be fixed with a non-breaking space (`&nbsp;` in HTML).
